### PR TITLE
Update: Toro 50283 KS Germany theme creation

### PR DESCRIPTION
**Pull request by muhammad-irtiza04**
Ticket: https://jira.tapestry.support/browse/TORO-50283

### CODE REVIEW

**Comment by p-musi** [comment]
Hi @muhammad-irtiza04,

We're not looking to copy the contents of the Japan theme over to a new German theme, we want the new German theme to be as close to the base as possible.

Screen Shot 2022-10-27 at 1 59 10 PM
Please create a new folder alongside the existing "japan" folder, and name is "germany".  Each theme inherits from the default, so you only need to add the files you want to override in that theme.

Then copy the file from "src/brands/kate-spade/default/font/face.json" over to "src/brands/kate-spade/germany/font/face.json" and update the font URLs to what I posted in my comment on the ticket.

**Comment by muhammad-irtiza04** [comment]
> Hi @muhammad-irtiza04,
> 
> We're not looking to copy the contents of the Japan theme over to a new German theme, we want the new German theme to be as close to the base as possible.
> 
> Screen Shot 2022-10-27 at 1 59 10 PM Please create a new folder alongside the existing "japan" folder, and name it "germany". Each theme inherits from the default, so you only need to add the files you want to override in that theme.
> 
> Then copy the file from "src/brands/kate-spade/default/font/face.json" over to "src/brands/kate-spade/germany/font/face.json" and update the font URLs to what I posted in my comment on the ticket. No other files besides this face.json are required in the germany folder at this time.

**Comment by muhammad-irtiza04** [comment]
I have made all the changes and push the code you can review it.

**Review by p-musi — CHANGES_REQUESTED** [review]
Hi @muhammad-irtiza04,

This is not correct.  Please refer to my PR where I implemented the germany theme for Coach.  Only one file should be added.

https://github.com/Tapestry-Inc/design-tokens/pull/169

File: src/brands/kate-spade/germany/font/face.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+  "font": {
+    "face": {
+      "1": {
+        "normal": {
+          "family": {
+            "value": "FuturaNowTextRegular(German)"
```
Comment (p-musi) [inline src/brands/kate-spade/germany/font/face.json:7]: Please ensure that this is not different from the default.

File: src/brands/kate-spade/germany/font/face.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            "value": "normal"
+          },
+          "weight": {
+            "value": "normal"
+          },
+          "stretch": {
+            "value": "normal"
+          },
+          "local": {
+            "value": ["Futura Now Text Regular (German)"]
```
Comment (p-musi) [inline src/brands/kate-spade/germany/font/face.json:19]: Please ensure that this is not different from the default.

File: src/brands/kate-spade/germany/font/face.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          },
+          "woff2": {
+            "value": "https://assets.katespade.com/de/fonts/FuturaNowTextRegular.woff2"
+          }
+        },
+        "medium": {
+          "family": {
+            "value": "FuturaNowHeadlineMedium(German)"
```
Comment (p-musi) [inline src/brands/kate-spade/germany/font/face.json:27]: Please ensure that this is not different from the default.

File: src/brands/kate-spade/germany/font/face.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            "value": "normal"
+          },
+          "weight": {
+            "value": "normal"
+          },
+          "stretch": {
+            "value": "normal"
+          },
+          "local": {
+            "value": ["Futura Now Headline Medium (German)"]
```
Comment (p-musi) [inline src/brands/kate-spade/germany/font/face.json:39]: Please ensure that this is not different from the default.

File: src/brands/kate-spade/germany/font/face.json [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+          },
+          "woff2": {
+            "value": "https://assets.katespade.com/de/fonts/FuturaNowHeadlineMedium.woff2"
+          }
+        }
+      },
+      "2": {
+        "normal": {
+          "family": {
+            "value": "ITCNewBaskervilleRoman(German)"
```
Comment (p-musi) [inline src/brands/kate-spade/germany/font/face.json:49]: Please ensure that this is not different from the default.

File: src/brands/kate-spade/germany/font/face.json [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+            "value": "normal"
+          },
+          "weight": {
+            "value": "normal"
+          },
+          "stretch": {
+            "value": "normal"
+          },
+          "local": {
+            "value": ["ITC New Baskerville Roman (German)"]
```
Comment (p-musi) [inline src/brands/kate-spade/germany/font/face.json:61]: Please ensure that this is not different from the default.

**Review by p-musi — CHANGES_REQUESTED** [review]
Hi @muhammad-irtiza04,

Looks mostly good now.  Please see review feedback comments on specific lines.  We don't want to have "(German)" appended to the font values.

File: src/brands/kate-spade/germany/font/face.json (hunk 6 above)
Comment (muhammad-irtiza04) [reply src/brands/kate-spade/germany/font/face.json]: Resolved: Now it's according to default

File: src/brands/kate-spade/germany/font/face.json (hunk 3 above)
Comment (muhammad-irtiza04) [reply src/brands/kate-spade/germany/font/face.json]: Resolved: Now it's according to default

File: src/brands/kate-spade/germany/font/face.json (hunk 2 above)
Comment (muhammad-irtiza04) [reply src/brands/kate-spade/germany/font/face.json]: Resolved: Now it's according to default

File: src/brands/kate-spade/germany/font/face.json (hunk 1 above)
Comment (muhammad-irtiza04) [reply src/brands/kate-spade/germany/font/face.json]: Resolved: Now it's according to default

**Review by ejaz-tapestry — APPROVED** [review]

**Review by p-musi — APPROVED** [review]

**Pull request merged**